{{cookiecutter.__src_folder_name}}/infra/main.bicep

```diff
@@ -47,6 +51,11 @@ module keyVault './core/security/keyvault.bicep' = {
     location: location
     tags: tags
     principalId: principalId
+    allowPublicNetworkAccess: false
+    networkAcls: {
+      allowAzureServicesAccess: allowAzureServicesAccess
+      defaultAction: defaultAction
+    }
```

Comment:
I don't know if it is needed to add this section here provide the params or if we just want it to fall back to the default. 
In terms of patterns is it better to put it in the main.bicep file that there are these options so its up front, or to stash them away?

Reply:
It's not required, remove it from this file

Reply:
OK cool have removed